class-jetpack-currencies.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * Jetpack_Currencies: Utils for displaying and managing currencies.
```

Comment:
Instead of another class file in the root of Jetpack, could this be a lib, stored in `_inc/lib/` (and with the other libs on WordPress.com) and usable via `jetpack_require_lib`?

I suppose this could even be a package, if we think others may benefit from this in the future?

Reply:
> could this be a lib

Sure! I wasn't sure about the best approach here, so I added to the root because didn't notice we use the `_inc/lib` folder for maintaining libraries. Moved there in 533ea293099a358c090e86076840de5e5745b6bd.

> I suppose this could even be a package

Yeah, that makes sense. I'm unfamiliar though about creating new packages, so I think we can leave that for a follow-up.